packages/app/components/home/home-slider.tsx

```diff
@@ -0,0 +1,16 @@
+import { FlatList, FlatListProps } from "react-native";
+
+type Props<T> = Pick<FlatListProps<T>, "data" | "renderItem"> & {
+  slidesPerView?: number;
+};
+export function HomeSlider<T>({ data, ...rest }: Props<T>) {
+  return (
+    <FlatList
```

Comment:
Don't use FlatList, even for the tiniest components it should be FlashList, since it improves rendering performance

Reply:
Actually, I have tried it. If you make two FlashLists on this page, the screen will freeze sometimes. You can just replace this to see.
But for this component, I'm plan to use ScrollView instead because we only have a maximum of 8 items.

Reply:
If true, better stick with FlatList. It's still better than ScrollView